prov/efa/src/efa_cq.c

```diff
@@ -312,7 +312,21 @@ int efa_cq_open(struct fid_domain *domain_fid, struct fi_cq_attr *attr,
 
 	init_attr_ex.cqe = attr->size ? attr->size : EFA_DEF_CQ_SIZE;
 
+#ifdef SUPPORT_EX_CQ
```

Comment:
`SUPPORT_EX_CQ` is too general. should be something like `HAVE_EFADV_CQ_EX`.

Reply:
Sounds good. Will address.